config/manager/default-config/deployment.yaml

```diff
@@ -99,6 +99,8 @@ spec:
           env:
             - name: APP_CONFIG_backend_listen_port
               value: "7007"
+            - name: NODE_OPTIONS
+              value: '--no-node-snapshot'
```

Comment:
Just curious, why this is not defined in the Dockerfile (https://github.com/janus-idp/backstage-showcase/blob/main/docker/Dockerfile and https://github.com/janus-idp/backstage-showcase/blob/main/.rhdh/docker/Dockerfile)? This would avoid having to set it in both the operator and the Helm Chart (https://github.com/redhat-developer/rhdh-chart/pull/14).

Reply:
Also, could this option have any effect on previous versions of RHDH not running on Node 20?

Reply:
> Also, could this option have any effect on previous versions of RHDH not running on Node 20?

Unlikely, as the chart/operator will deploy a new RHDH container image, which bundles newer plugins.

Reply:
I'm ok with this change but I agree, if we can fix it in the hub container instead of BOTH the chart and operator, that would seem cleaner.

Reply:
I'll make a PR in the showcase to update the dockerfile.

Reply:
https://github.com/janus-idp/backstage-showcase/pull/1277